arm/Microsoft.Network/azureFirewalls/readme.md

```diff
@@ -8,28 +8,33 @@ This module deploys a firewall.
 - [Parameters](#Parameters)
 - [Outputs](#Outputs)
 - [Considerations](#Considerations)
+- [Template references](#Template-references)
```

Comment:
Please merge the latest main into your branch and re-generate the Readme. The `TemplateReferences` table should be gone.